ukat/mapping/b0.py

```diff
@@ -94,6 +94,17 @@ def __init__(self, pixel_array, echo_list, affine, mask=None,
             self.phase_difference = self.phase1 - self.phase0
             # B0 Map calculation
             self.b0_map = self.phase_difference / (2 * np.pi * self.delta_te)
+            # Check if the full cycle exists and if so, correct the output
+            # B0 Map based on the dynamic range
+            central_pixel_value = np.mean(self.b0_map[int(self.shape[0] / 2),
+                                                      int(self.shape[1] / 2),
+                                                      ...])
+            dynamic_range = 1 / self.delta_te
+            if np.abs(central_pixel_value) > dynamic_range / 2:
```

Comment:
Could you do something along the lines of 
```python
self.b0_map -= (central_pixel_value // dynamic_range) * dynamic_range
```
to replace 103 - 107. It's more succinct and can deal with cases where central_pixel_value is more than one multiple of dynamic_range away from zero.